app/code/core/Mage/Core/Helper/Abstract.php

```diff
@@ -42,6 +42,10 @@ abstract class Mage_Core_Helper_Abstract
      */
     protected $_layout;
 
+    protected array $modulesDisabled = [];
```

Comment:
Traditionally, protected var has prefix '_'. Do we want to break tradition?

Reply:
i'd not stick with old traditions.

i think we had PRs that added protected properties w/o underscore.